setup.cfg

```diff
@@ -22,6 +22,7 @@ spacy_factories =
 
 spacy_architectures =
     curated-transformers.BertTransformer.v1 = curated_transformers.models:build_bert_transformer_model_v1
+    curated-transformers.RobertaTransformer.v1 = curated_transformers.models:build_roberta_transformer_model_v1
```

Comment:
Is there any particular reason why we don't need to/aren't adding the various piece encoder entry points to this section too?